pkg/kubectl/cmd/apply.go

```diff
@@ -607,7 +606,9 @@ func (p *patcher) patchSimple(obj runtime.Object, modified []byte, source, names
 			return nil, nil, cmdutil.AddSourceToErr(fmt.Sprintf(createPatchErrFormat, original, modified, current), source, err)
 		}
 	}
-
+	if string(patch) == "{}" {
+		return patch, obj, nil
```

Comment:
It would be nice to output a different message in this case. Rather than:
```
pod "pause" configured
```
Something like:
```
no changes required for pod "pause"
```